pkg/registry/apps/controllerrevision/strategy_test.go

```diff
@@ -127,55 +125,6 @@ func TestStrategy_ValidateUpdate(t *testing.T) {
 	}
 }
 
-func TestControllerRevisionToSelectableFields(t *testing.T) {
-	rev := newControllerRevision("validname", "validns", newObject(), 0)
-	fieldSet := ControllerRevisionToSelectableFields(rev)
-	if fieldSet.Get("metadata.name") != rev.Name {
-		t.Errorf("expeted %s found %s", rev.Name, fieldSet.Get("metadata.name"))
-	}
-	if fieldSet.Get("metadata.namespace") != rev.Namespace {
-		t.Errorf("expeted %s found %s", rev.Namespace, fieldSet.Get("metadata.namespace"))
-	}
-}
-
-func TestGetAttrs(t *testing.T) {
-	rev := newControllerRevision("validname", "validns", newObject(), 0)
-	labelSet, fieldSet, uninitialized, err := GetAttrs(rev)
-	if err != nil {
-		t.Fatal(err)
-	}
-	if uninitialized {
-		t.Errorf("unexpected attrs")
-	}
-	if fieldSet.Get("metadata.name") != rev.Name {
-		t.Errorf("expeted %s found %s", rev.Name, fieldSet.Get("metadata.name"))
-	}
-	if fieldSet.Get("metadata.namespace") != rev.Namespace {
-		t.Errorf("expeted %s found %s", rev.Namespace, fieldSet.Get("metadata.namespace"))
-	}
-	if labelSet.Get("foo") != rev.Labels["foo"] {
-		t.Errorf("expected %s found %s", rev.Labels["foo"], labelSet.Get("foo"))
-	}
-}
-
-func TestMatchControllerRevision(t *testing.T) {
```

Comment:
This entire method was removed.  I don't see a reason to re-test the stock selector.